### DISCUSSION TITLE
nfdump_influx for InfluxDB

### DISCUSSION THREAD

**Discussion by phaag** [phaag/nfdump#311] [Announcements]
In order to play and compare systems, I added nfdump_influx for InfluxDB. As the Prometheus exporter, this is playware and not intended production. It's interface and the metrics exported may change any time.
It's purpose is to see if those exporters are useful.

Feel free to join the discussion, opened on Github.

https://github.com/phaag/nfdump_influx

**Comment by ipksupport** [comment]
I am working with nfdump to export data to influx format (not realtime import into influx, just creating the influxdb file for import) and trying to aggregate the flow info to reduce the cardinality of the data(as influx does have limits here). I would like to add some V9/10 tags according to the AppFlow Info (IPFIX) - V10 templatesTemplate ID - 611 as per documentation here: [AppFlow](https://docs.citrix.com/en-us/citrix-sd-wan-orchestrator/appflow-ipfix.html) and here [IPFIX iana](https://www.iana.org/assignments/ipfix/ipfix.xhtml). 

Mainly the Application ID stuff in "AppFlow Template 612", 94,95 and 96. Is there a way to easily add this to nfcapd (or does this needs to be compiled in)? Also VRF ID -ingressVRFID, 234 would be nice to have.
I am happy to share my code so far, or if you can give a pointer as where to add this in the nfcapd code (have not dived into that yet).
Bert-Jan Kamp

**Comment by phaag** [comment]
The tags are integrated meanwhile.
